src/Components/UI/LazyLoadImages/LazyLoadImages.jsx

```diff
@@ -0,0 +1,13 @@
+import LazyLoad from 'react-lazyload';
+
+export default function LazyLoadImages({ src, altText, classText }) {
```

Comment:
Le agregaría un default value por si no necesitan pasarle la clase!

```suggestion
export default function LazyLoadImages({ src, altText, classText = '' }) {
```